src/www/containers/userprofile/UserProfile.scss

```
@@ -0,0 +1,350 @@
+      margin: 0;
+      padding: 0;
+
+      > li {
+        cursor: pointer;
+        padding: 0.5rem;
+        padding-top: 0;
+
+        &:hover {
+          border-bottom: 2px solid orangered;
+        }
+
+        &:not(:first-of-type) {
+          margin-left: 4rem;
+        }
+
+        &.Active-tab {
+          border-bottom: 2px solid red;
+          font-weight: bold;
+        }
+      }
+    }
+
+    .Content {
+      margin-top: 1rem;
+
+      > :not(.active) {
+        display: none;
+      }
+
+      .Actions,
+      .Teams {
+        > div {
+          background-color: rgba($color: white, $alpha: 0.3);
+          border: 1px solid rgba($color: black, $alpha: 0.4);
+          border-radius: 0.5rem;
+          box-shadow: 0 0 5px 1px rgba($color: black, $alpha: 0.2);
+          display: flex;
+          flex-direction: column;
+          padding: 0rem 2rem;
+
+          &:not(:first-of-type) {
+            margin-top: 1rem;
+          }
+
+          > h3 {
+            font-size: 1.5rem;
+            margin-bottom: 1rem;
+            margin-top: 2rem;
+            text-align: left;
+          }
+
+          > ul {
+            list-style-type: none;
+            padding: 0;
+            text-align: left;
+
+            > li:not(:first-of-type) {
+              margin-top: 0.8rem;
+            }
+          }
+
+          > p {
+            align-self: center;
+            cursor: pointer;
+            padding: 0rem;
+            text-decoration-line: underline;
+            width: fit-content;
+          }
+
+          .Card {
+            display: flex;
+            flex-direction: row;
+
+            .Card-image {
+              border-radius: 0.5rem;
+              height: 10rem;
+              width: 10rem;
+            }
+
+            .Card-body {
+              display: flex;
+              flex-direction: column;
+              justify-content: space-between;
+              margin: 0.75rem 1.5rem;
+
+              h4 {
+                margin: 0;
+              }
+
+              .Card-categories {
+                align-items: center;
+                display: flex;
+                flex-direction: row;
+
+                > ul {
+                  display: flex;
+                  flex-direction: row;
+                  list-style-type: none;
+                  padding-left: 0.5rem;
+
+                  > li {
+                    border: 1px solid rgba($color: black, $alpha: 0.3);
+                    border-radius: 0.2rem;
+                    padding: 0.25rem;
+
+                    &:not(:first-of-type) {
+                      margin-left: 0.5rem;
+                    }
+                  }
+                }
+              }
+            }
+          }
+
+          .Link {
+            align-self: center;
+            cursor: pointer;
+            padding: 0rem;
+            width: fit-content;
+            color: black;
+          }
+        }
+      }
+    }
+  }
+}
+
+@media only screen and(max-width: 1024px) {
```

I think this is kicking in too late, wasting too much horizontal space.
What do you think about this:
```suggestion
@media only screen and(max-width: 1275px) {
```